test/infrastructure/container/docker.go

```diff
@@ -341,28 +341,6 @@ func dockerContainerToContainer(container *types.Container) Container {
 	}
 }
 
```

Comment:
Those methods were unused, both also exist as funcs which are used.

Reply:
Oops, I must have done that while moving things around. Thanks for cleaning that up!

Reply:
Thought so, you're welcome :)